build/build.go

```diff
@@ -129,3 +141,75 @@ func configureScript(script *exec.Cmd, extension core.Extension) error {
 }
 
 const rwxr_xr_x = 0755
+
+func setLocalization(extApi *api.ExtensionsApi, extension *core.Extension) error {
+	//HACK: for debugging / output to console, related to: (https://github.com/Shopify/checkout-web/issues/9581)
+	fmt.Println("setLocalization() called...")
+	localization, err := api.GetLocalization(extension)
+
+	if err != nil {
+		log.Println(err)
+		return err
+	}
+	extension.Localization = localization
+
+	//HACK: for debugging / output to console, related to: (https://github.com/Shopify/checkout-web/issues/9581)
+	jsonBytes, err := json.Marshal(extension)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	fmt.Println(string(jsonBytes))
+
+	extension.Development.Status = "success"
+	extApi.Notify([]core.Extension{*extension})
```

Comment:
This also can be removed if we make the main WatchLocalization responsible for calling `report`